internal/provider/resources/service_account.go

```diff
@@ -40,11 +47,12 @@ type ServiceAccountResourceModel struct {
 	AccountID       customtypes.UUIDValue `tfsdk:"account_id"`
 	AccountRoleName types.String          `tfsdk:"account_role_name"`
 
-	APIKeyID         types.String               `tfsdk:"api_key_id"`
-	APIKeyName       types.String               `tfsdk:"api_key_name"`
-	APIKeyCreated    customtypes.TimestampValue `tfsdk:"api_key_created"`
-	APIKeyExpiration customtypes.TimestampValue `tfsdk:"api_key_expiration"`
-	APIKey           types.String               `tfsdk:"api_key"`
+	APIKeyID               types.String               `tfsdk:"api_key_id"`
+	APIKeyName             types.String               `tfsdk:"api_key_name"`
+	APIKeyCreated          customtypes.TimestampValue `tfsdk:"api_key_created"`
+	APIKeyExpiration       customtypes.TimestampValue `tfsdk:"api_key_expiration"`
+	OldKeyExpiresInSeconds types.Int32                `tfsdk:"old_key_expires_in_seconds"`
```

Comment:
if not provided, does `types.Int32` default to `0?

Reply:
i guess it would depend on the behavior of `.ValueInt32()`

Reply:
Yeah exactly:

<img width="702" alt="image" src="https://github.com/user-attachments/assets/1787fd4a-5457-4aab-9dde-28b83a35a824">

We also [set a default of 0](https://github.com/PrefectHQ/terraform-provider-prefect/blob/5264f2d06f9b3b83740378b1237dcd7419842230/internal/provider/resources/service_account.go#L167).